MCPForUnity/UnityMcpServer~/src/tools/manage_prefabs.py

```
@@ -28,8 +24,11 @@ def manage_prefabs(
                                "Allow replacing an existing prefab at the same path"] | None = None,
     search_inactive: Annotated[bool,
                                "Include inactive objects when resolving the target name"] | None = None,
+    component_properties: Annotated[str, "Component properties in JSON format"] | None = None,
```

Ruff (0.14.3)

Unused function argument: `component_properties`

(ARG001)